setup.py

```diff
@@ -34,33 +34,26 @@
 # limitations under the License.
 # ============================================================================
 #
-import setuptools
+from pathlib             import Path
+from pyTooling.Packaging import DescribePythonPackageHostedOnGitHub
 
-with open("README.md", "r") as file:
-	long_description = file.read()
+gitHubNamespace =        "edaa-org"
+packageName =            "pyEDAA.IPXACT"
+packageDirectory =       packageName.replace(".", "/")
+packageInformationFile = Path(f"{packageDirectory}/__init__.py")
 
-setuptools.setup(
-	name="pyIPXACT",
-	version="0.1.6",
-	author="Patrick Lehmann",
-	author_email="Paebbels@gmail.com",
+DescribePythonPackageHostedOnGitHub(
+	packageName=packageName,
 	description="A Document-Object-Model (DOM) for IP-XACT files.",
-	long_description=long_description,
-	long_description_content_type="text/markdown",
-	url="https://github.com/Paebbels/pyIPXACT",
-	packages=setuptools.find_packages(),
+	gitHubNamespace=gitHubNamespace,
+	sourceFileWithVersion=packageInformationFile,
+	developmentStatus="pre-alpha",
 	classifiers=[
```

Comment:
Oh I wasn't precise enough - sorry.

Actually none of the classifiers is needed:
https://github.com/pyTooling/pyTooling.Packaging/blob/main/pyTooling/Packaging/__init__.py?ts=2#L257

*pre-alpha is added by `developmentStatus`.

Reply:
Done.